### ISSUE TITLE
PMP has trouble finding content when unauthenticated

### ISSUE THREAD

**Issue by bouscram** [plexinc/plex-media-player#798]
## How to file an issue - REMOVE THIS SECTION
 - Add a prefix to the title scoping the issue, e.g. `gui: `, `player: ` ... 
 - use a title that is clear enough
 - If appropriate, add a topic label to group the issue with a larger effort and allow filtering by that topic.

### Test environment
**PMS Version:**
**PMP Version:**
**Platform (Windows/OSX/Embedded):**

### Steps to reproduce
1.

### Current behavior

1.

### Expected behavior

1.

**Comment by mseeley** [comment]
Thanks @bouscram. When launching PMP with the server unclaimed, is the server running on the same machine as PMP? I believe it must because you mentioned PMP is unauthenticated.

I've tagged our TV team to look into the infinite spinner regression.

**Comment by bouscram** [comment]
Yes, PMP and PMS on the same machine. The problem seems to be present in both UIs, only there are ways to make it disappear with the web UI (by changing the language or scanning a library).

I don't have time right now to test extensively, but I'm pretty sure I have seen a similar behaviour when on different machines (with PMP authenticated - because it insists on doing so while it really shouldn't, but this is another problem) - but I can't be positive...

**Comment by mseeley** [comment]
Thank you @bouscram, we'll get some investigation into this. 👍

**Comment by mseeley** [comment]
I'll take a look at this and tag in a TV developer if necessary.

**Comment by mseeley** [comment]
Thanks Chris for confirming and communicating that the desktop layout fix will be in version 3.73.4.

Over to TV for debugging before closing.

**Comment by bouscram** [comment]
Confirming the bug is corrected for Web UI in version 2.23 (Web UI 3.77.2), but not in TV mode
